easybuild/easyblocks/g/gromacs.py

```diff
@@ -201,14 +238,54 @@ def configure_step(self):
                 run_cmd(plumed_cmd, log_all=True, simple=True)
 
         else:
+            if '-DGMX_MPI=ON' in self.cfg['configopts']:
+                mpi_numprocs = self.cfg.get('mpi_numprocs', 0)
+                if mpi_numprocs == 0:
+                    self.log.info("No number of test MPI tasks specified -- using default: %s",
+                                  self.cfg['parallel'])
+                    mpi_numprocs = self.cfg['parallel']
+
+                elif mpi_numprocs > self.cfg['parallel']:
+                    self.log.warning("Number of test MPI tasks (%s) is greater than value for 'parallel': %s",
+                                     mpi_numprocs, self.cfg['parallel'])
+
+                mpiexec = self.cfg.get('mpiexec')
+                if mpiexec:
+                    mpiexec = which(mpiexec)
```

Comment:
better rename `mpiexec = ` to `mpiexec_path = ` here, so you can reuse `mpiexec` below?

Reply:
Done